app/models/geographic_item.rb

```diff
@@ -1067,7 +1086,7 @@ def intersects?(target_geo_object)
   # @param [geo_object]
   # @return [Boolean]
   def distance?(target_geo_object)
-    self.geo_object.distance?(target_geo_object)
```

Comment:
Was this a uncaught error?

Reply:
Yes, undefined method error. But I think this method is not actually being used (but I was hoping it was useful for me, it was not).

Reply:
Go ahead and remove the method if it is not used.

Reply:
Other than `expect(geographic_item).to respond_to(:distance?)` doesn't seem to be used anywhere (and spec was only testing the method exists without actually executing it). This "go ahead" is for the entire PR or just removing this method?

Reply:
Remove it, and expectionation.